iati/tests/test_validator.py

```diff
@@ -355,52 +360,77 @@ class ValidateCodelistsBase(ValidationTestBase):
 
     @pytest.fixture
     def schema_version(self):
-        """Return an Activity Schema with the Version Codelist added."""
-        schema = iati.default.activity_schema(None, False)
-        codelist = iati.default.codelist('Version')
+        """Return an Activity Schema with the Version Codelist added.
+
+        Todo:
+            Stop this being fixed to 2.02.
+
+        """
+        schema = iati.default.activity_schema('2.02', False)
+        codelist = iati.default.codelist('Version', '2.02')
 
         schema.codelists.add(codelist)
 
         return schema
 
     @pytest.fixture
     def schema_org_type(self):
-        """Return an Activity Schema with the OrganisationType Codelist added."""
-        schema = iati.default.activity_schema(None, False)
-        codelist = iati.default.codelist('OrganisationType')
+        """Return an Activity Schema with the OrganisationType Codelist added.
+
+        Todo:
+            Stop this being fixed to 2.02.
+
+        """
+        schema = iati.default.activity_schema('2.02', False)
+        codelist = iati.default.codelist('OrganisationType', '2.02')
 
         schema.codelists.add(codelist)
 
         return schema
 
     @pytest.fixture
     def schema_incomplete_codelist(self):
-        """Return an Activity Schema with an incomplete Codelist added."""
-        schema = iati.default.activity_schema(None, False)
-        codelist = iati.default.codelist('Country')
+        """Return an Activity Schema with an incomplete Codelist added.
```

Comment:
Worth parameterising this fixture to all supported pyIATI versions? Same in other places where `Stop this being fixed to 2.02.` is added as a Todo.

Reply:
The whole `Stop this being fixed to 2.02.` needs looking at separately based on a wider test strategy. I'd suggest that it'd be better to sort it out completely rather than patching it up and hoping that all is OK.

This could be added as a checklist item in #223, or an Issue created (if there's not one already).